UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.cs

```diff
@@ -34,7 +34,7 @@ public partial class SubSceneManager : MonoBehaviour
 
 	public bool IsMaintRooms
 	{
-		get { return serverChosenAwaySite == "Backrooms"; }
+		get { return serverChosenAwaySite == "AwaySites/Backrooms.json"; }
```

Comment:
why is this hardcoded?
terrible, bad, you should feel bad

Reply:
well notmyfault the original implementation was Hardcoded